policy/global_tunables

```diff
@@ -32,6 +32,13 @@ gen_tunable(allow_execmod,false)
 ## </desc>
 gen_tunable(allow_execstack,false)
 
+## <desc>
+## <p>
+## Allow raw memory device (/dev/mem, /dev/kmem, /dev/mergemem, dev/oldmem, /dev/port) access for unconfined executables.
```

Comment:
These domains are confined, not unconfined.

Reply:
Fixed